r/NEWS.md

```diff
@@ -16,6 +16,12 @@
   specific language governing permissions and limitations
   under the License.
 -->
+# development version
+
+  * `lubridate::parse_date_time()` datetime parser (only for year, month, and day components with separators). Not all functionality has been implemented:
+    * the `orders` argument in the Arrow binding works slightly different than in `lubridate::parse_date_time()` and closer to `lubridate::parse_date_time2()` and `lubridate::fast_strptime()`: `orders` are transformed into `formats` which subsequently get applied in turn. There is no `select_formats` parameter and no inference takes place. `lubridate::parse_date_time()` usually "trains" formats on a subset of the input vector and then applies them according to performance on the training set. 
```

Comment:
I find this bullet point a bit confusing (especially if read outside the context of this PR):

I know the history, but as an end-user I would immediately wonder: if the implementation "if "parse_date_time()` is closer to `parse_date_time2()` or `fast_strptime()`, why didn't they implement those instead?"

Perhaps a better approach would be to enumerate positively what _can_ be done currently rather than referencing other functions (+ listing what can't)

Reply:
re-worded. hopefully it is a bit clearer.